controllers/user.js

```diff
@@ -0,0 +1,74 @@
+const {User} = require('../models/')
+class ControllerUser {
+    static getAllUser(req,res){
+        User
+        .findAll()
+        .then(users=>{
+            res.status(200).json(users)
+        })
+        .catch(err=>{
+            res.status(500).json(err)
+        })        
+    }
+
+    static getOne(req,res){
+        User
+        .findByPk(req.params.id)
+        .then(user=>{
```

Comment:
Kasih spasi diantara arrow function

```suggestion
        .then(user => {
```